homeassistant/components/qnap/const.py

```diff
@@ -0,0 +1,57 @@
+"""The Qnap constants."""
+from homeassistant.const import (
+    DATA_GIBIBYTES,
+    DATA_RATE_MEBIBYTES_PER_SECOND,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+)
+
+CONF_DRIVES = "drives"
+CONF_NICS = "nics"
+CONF_VOLUMES = "volumes"
+COMPONENTS = ["sensor"]
+DEFAULT_NAME = "Qnap"
+DEFAULT_PORT = 8080
+DEFAULT_TIMEOUT = 5
+DOMAIN = "qnap"
+
+
+NOTIFICATION_ID = "qnap_notification"
+NOTIFICATION_TITLE = "QNAP Sensor Setup"
+
+_SYSTEM_MON_COND = {
+    "status": ["Status", None, "mdi:checkbox-marked-circle-outline"],
```

Comment:
Add a fourth entry to each of these sensor info: the value for `entity_registry_enabled_default` property.

Return `False` if the entity should be disabled by default and users need to enable it .